### PR TITLE
Remove readOnly from nextLink (#418)

### PR DESCRIPTION
**Pull request by markcowl**
Fix for #418

### CODE REVIEW

File: package.json
Comment (timotheeguerin) [inline package.json:1]: are those changes on purpose, I think it will complain that it doesn't match the version in core repo?

File: pnpm-lock.yaml
Comment (timotheeguerin) [inline pnpm-lock.yaml:1]: probably revert this file too

File: package.json
Comment (markcowl) [reply package.json:1]: this was just from running pnpm update.  Should I update core at the same time?

Comment (timotheeguerin) [reply package.json:1]: I would just revert here, don't think it brings much value to update right now

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**